src/views/passwordReset.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { ToastContainer } from 'react-toastify';
+import { Form, FormControl, Button, Container } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import NavigationBar from '../components/homeNavbar';
+import resetPasswordLinkAction from '../redux/actions/passwordreset.action';
+
+
+
+class PasswordResetForm extends Component {
+  state = {
+    successmessage: '',
+    errormessage: ''
+  };
+
+  handleChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value
+    }
+    );
+  };
+
+  handleSubmit = (e) => {
+    const { resetPasswordLinkAction } = this.props;
+    e.preventDefault();
+    resetPasswordLinkAction(this.state);
+  }
+  
+  render() { 
+    return (
+      <div>
+        <NavigationBar />
+        <ToastContainer />
+        <Container className="reset">
+          <div>
+            <h1 className="resetmessage">
+        First, let&apos;s find your account
+            </h1>
+          </div>
+          <Form onSubmit={this.handleSubmit}>
+            <FormControl name="email" placeholder="Please enter your email" onChange={this.handleChange} className="reset-input" required />
+            <div className="reset-buttons">
+              <Button variant="outline-danger" type="reset" className="reset-buttons">Cancel</Button>
+              <Button variant="primary" type="submit" className="reset-buttons">Find account</Button>
+            </div>
+          </Form>
+        </Container>
+        
+      </div>
+    );
+  }
+}
+
+PasswordResetForm.propTypes = {
+  resetPasswordLinkAction: PropTypes.func.isRequired,
+};
+
+const mapStatetoProps = state => ({
+  error: state.resetpasswordlink.error,
```

Comment:
Hello @IssaIan ,
This is good work. What do you think about using destructuring on the state instead? I think it would make it more elegant

Reply:
@Ogutu-Brian I don't understand how this would be done here. I think this is already destructured as it is.

Reply:
```
const mapStatetoProps=({resetPasswordLink})=>({
    error:resetPasswordLink.error
})
```
A state is an object, so you can always access its content through destructuring